src/components/ProductCard/ProductCard.tsx

```diff
@@ -1,7 +1,15 @@
+import React from 'react';
 import './ProductCard.scss';
 import imgSrc from 'assets/images/product-card/product-1.png';
+import { Phone } from 'app/types/Phone';
+
+type Props = {
+  phoneCard: Phone;
+};
+
+export const ProductCard: React.FC<Props> = ({ phoneCard }) => {
+  const { name, fullPrice, price, screen, capacity, ram } = phoneCard;
```

Comment:
![image](https://github.com/fe-may23-teamscript/ecommerce_frontend/assets/101696452/4b0c7f0e-cec2-4437-be3f-0420f0eab4b5)
1) By Mockup on Tablets and Phones, cards should have "collision" with the right border of their container
![image](https://github.com/fe-may23-teamscript/ecommerce_frontend/assets/101696452/5b646378-eaed-4d5d-b748-47628c95036a)
2) The horizontal distance between cards should be smaller on the tablet version
3) The text color on "Add to cart" button should be white
4) The card size should be bigger than is now because on the mockup they got 5fr from the page grid
![image](https://github.com/fe-may23-teamscript/ecommerce_frontend/assets/101696452/7f6725f0-2471-4d69-8f47-a97f6c6db4ec)